### ISSUE TITLE
flag.domains with mailchecker at build-time

### ISSUE THREAD

**Issue by FGRibreau** [FGRibreau/mailchecker#427]
https://dl.red.flag.domains/red.flag.domains_fr.txt

**Comment by YasharF** [comment]
Just a thought, should this be during the build-time, install time, startup time, or a combination (i.e. have fallbacks)?  By moving the data pull closer to the startup/runtime the consumers can have more up-to-date lists without having to constantly download/upgrade the package.

**Comment by FGRibreau** [comment]
Good point, maybe it should be at install time by default and optionally activated at run-time I think.

The download URL should point to https://github.com/FGRibreau/mailchecker/blob/master/list.txt and we could leverage github actions to automatically update that lists and merge it with https://dl.red.flag.domains/red.flag.domains_fr.txt ?

If that's something you are interested in doing, I'm open for a PR :)

**Comment by FGRibreau** [comment]
>  there might be value in implementing a dynamic lookup feature instead

Indeed, I'm open for a PR. This feature should be disabled by default

> However, integration with abuseipdb might fall outside the scope of mailchecker and could potentially be part of a different package.

Exactly, that's for something else
